molo/core/tests/test_views.py

```diff
@@ -141,6 +141,10 @@ def test_able_to_copy_main(self):
             title='banner', slug='banner', banner_link_page=article)
         self.banner_index.add_child(instance=banner)
         banner.save_revision().publish()
+        banner2 = BannerPage(
+            title='banner2', slug='banner2', banner_link_page=self.yourmind)
+        self.banner_index.add_child(instance=banner2)
+        banner2.save_revision().publish()
```

Comment:
@saeedmb8 I think it would be better to write  a test in `test_utils`, with specific reference to this functionality, rather than creating an instance where it throws an error when testing the view.